app/controllers/users_controller.rb

```diff
@@ -3,4 +3,15 @@ def show
     @user = current_user
     @user_characters = @user.characters.order(:name).page(params[:page]).per(3)
   end
+
+  def profile
+    respond_to do |format|
+      if User.exists?(params[:id])
+        @user = User.find(params[:id])
```

Comment:
@zzokki81 this still is a potential security issue. What happens when I call `users/123/profile` will I get all the info about any user? I shouldn't be able to do that. I should only be able to see my own data.